Source/SwiftLintFramework/Rules/ProtocolPropertyAccessorsOrderRule.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ProtocolPropertyAccessorsOrderRule.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 15/05/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct ProtocolPropertyAccessorsOrderRule: ConfigurationProviderRule, CorrectableRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "protocol_property_accessors_order",
+        name: "Protocol Property Accessors Order",
+        description: "When declaring properties in protocols, the order of accessors should be `get set`.",
+        nonTriggeringExamples: [
+            "protocol Foo {\n var bar: String { get set }\n }",
+            "protocol Foo {\n var bar: String { get }\n }",
+            "protocol Foo {\n var bar: String { set }\n }"
+        ],
+        triggeringExamples: [
+            "protocol Foo {\n var bar: String { ↓set get }\n }"
+        ],
+        corrections: [
+            "protocol Foo {\n var bar: String { ↓set get }\n }":
+                "protocol Foo {\n var bar: String { get set }\n }"
+        ]
+    )
+
+    public func validate(file: File) -> [StyleViolation] {
+        return violationRanges(file: file).map {
+            StyleViolation(ruleDescription: type(of: self).description,
+                           severity: configuration.severity,
+                           location: Location(file: file, characterOffset: $0.location))
+        }
+    }
+
+    private func violationRanges(file: File) -> [NSRange] {
+        return file.match(pattern: "\\bset\\s*get\\b", with: [.keyword, .keyword])
```

Comment:
this is pretty dumb because AFAIK the only place where `set get` is valid is inside a protocol property, so I didn't even bother to check if it was inside a property (and inside a protocol)